models/intermediate/shopify__orders__order_line_aggregates.sql

```diff
@@ -1,3 +1,5 @@
+{{ config(materialized='table') }}
```

Comment:
made this a table since it pulls from many large upstream tables, and is also referenced in several downstream tables.